src/charmed_kubeflow_chisme/exceptions/_error_status_with_message.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+class GenericCharmRuntimeError(Exception):
+    """Raised when the unit should be in ErrorStatus with a message to show in juju status.
+
+    This exception can be used in the charm code to indicate that there is an issue in runtime
+    caused by any type of error.
+
+    A typical usage might be:
+
+    ```python
+    ...
+    from charmed_kubeflow_chisme.exceptions import GenericCharmRuntimeError
+    ...
```

Comment:
```suggestion
    ```python
    from charmed_kubeflow_chisme.exceptions import GenericCharmRuntimeError
```
can save a couple of lines :) in case somebody reads doc strings from console

Reply:
but up to you :)